core/container/src/main/java/org/wildfly/swarm/Swarm.java

```diff
@@ -464,7 +464,13 @@ private static boolean isFatJar() throws IOException {
 
     private void loadStageConfiguration(URL url) {
         List<ProjectStage> projectStages = new ProjectStageFactory().loadStages(url);
-        String stageName = System.getProperty(SwarmProperties.PROJECT_STAGE, "default");
+        String stageName = System.getProperty(SwarmProperties.PROJECT_STAGE);
```

Comment:
Thank you for your feedback, but I believe that the if/else approach is good in this case :+1: